.github/workflows/ci.yml

```diff
@@ -531,6 +525,9 @@ jobs:
         env:
           FLUENT_IMAGE_TAG: ${{ matrix.image-tag }}
 
+      - name: Cleanup previous docker containers
```

Comment:
Nitpick but "Clean up" is the verb (two words), while cleanup is the noun (1 word).